pghrep/src/fmtutils/utils.go

```diff
@@ -0,0 +1,90 @@
+package fmtutils
+
+import (
+    "strings"
+    "strconv"
+    "math"
+)
+
+func RoundUp(input float64, places int) (newVal float64) {
+     var round float64
+     pow := math.Pow(10, float64(places))
+     digit := pow * input
+     round = math.Ceil(digit)
+     newVal = round / pow
+     return
+}
+
+func ByteFormat(inputNum float64, precision int) string {
+    var unit string
+    var returnVal float64
+
+    if inputNum >= 1000000000000000000000000 {
+        returnVal = RoundUp((inputNum / 1208925819614629174706176), precision)
+        unit = " YB" // yottabyte
+    } else if inputNum >= 1000000000000000000000 {
+        returnVal = RoundUp((inputNum / 1180591620717411303424), precision)
+        unit = " ZB" // zettabyte
+    } else if inputNum >= 10000000000000000000 {
+        returnVal = RoundUp((inputNum / 1152921504606846976), precision)
+        unit = " EB" // exabyte
+    } else if inputNum >= 1000000000000000 {
+        returnVal = RoundUp((inputNum / 1125899906842624), precision)
+        unit = " PB" // petabyte
+    } else if inputNum >= 1000000000000 {
+        returnVal = RoundUp((inputNum / 1099511627776), precision)
+        unit = " TB" // terrabyte
+    } else if inputNum >= 1000000000 {
+        returnVal = RoundUp((inputNum / 1073741824), precision)
+        unit = " GB" // gigabyte
+    } else if inputNum >= 1000000 {
+        returnVal = RoundUp((inputNum / 1048576), precision)
+        unit = " MB" // megabyte
```

Comment:
let's be more precise here – in Postgres, "MB" means mebibyte, not megabyte. It should be "MiB", but I don't think Postgres will fix the abbreviation soon. In our documents (including comments), let's be precise -- let's use emphasize that it's mebibytes, not megabytes

https://en.wikipedia.org/wiki/Mebibyte
https://en.wikipedia.org/wiki/Megabyte

It applies to other units as well.